mirdata/datasets/irmas.py

```diff
@@ -0,0 +1,251 @@
+        filename='IRMAS-TestingData-Part1.zip',
+        url='https://zenodo.org/record/1290750/files/IRMAS-TestingData-Part1.zip?download=1',
+        checksum='5a2e65520dcedada565dff2050bb2a56',  # the md5 checksum
+        destination_dir=None,  # relative path for where to unzip the data, or None
+    ),
+    'testing_data_2': download_utils.RemoteFileMetadata(
+        filename='IRMAS-TestingData-Part2.zip',
+        url='https://zenodo.org/record/1290750/files/IRMAS-TestingData-Part2.zip?download=1',
+        checksum='afb0c8ea92f34ee653693106be95c895',  # the md5 checksum
+        destination_dir=None,  # relative path for where to unzip the data, or None
+    ),
+    'testing_data_3': download_utils.RemoteFileMetadata(
+        filename='IRMAS-TestingData-Part3.zip',
+        url='https://zenodo.org/record/1290750/files/IRMAS-TestingData-Part3.zip?download=1',
+        checksum='9b3fb2d0c89cdc98037121c25bd5b556',  # the md5 checksum
+        destination_dir=None,  # relative path for where to unzip the data, or None
+    ),
+}
+
+
+DATA = utils.LargeData(
+    'irmas_index.json'
+)
+
+
+INST_DICT = {
+    'cel', 'cla', 'flu', 'gac', 'gel', 'org', 'pia', 'sax', 'tru', 'vio', 'voi'
+}
+
+
+class Track(core.Track):
+    """IRMAS track class
+
+    Args:
+        track_id (str): track id of the track
+        data_home (str): Local path where the dataset is stored.
+            If `None`, looks for the data in the default directory, `~/mir_datasets/Mridangam-Stroke`
+
+    Attributes:
+        track_id (str): track id
+        predominant_instrument (list): Training tracks predominant instrument
+        train (bool): flag to identify if the track is from the training of the testing dataset
+        genre (str): string containing the namecode of the genre of the track.
+        drum (bool): flag to identify if the track contains drums or not.
+    """
+
+    def __init__(self, track_id, data_home):
+        if track_id not in DATA.index:
+            raise ValueError('{} is not a valid track ID in Example'.format(track_id))
+
+        self.track_id = track_id
+
+        self._data_home = data_home
+        self._track_paths = DATA.index[track_id]
+        self.audio_path = os.path.join(self._data_home, self._track_paths['audio'][0])
+        self.annotation_path = os.path.join(
+            self._data_home, self._track_paths['annotation'][0]
+        )
+
+        # Dataset attributes
+        self.predominant_instrument = None
+        self.genre = None
+        self.drum = None
+        self.train = True
+
+        # TRAINING TRACKS
+        if '__' in track_id:
+            self.predominant_instrument = os.path.basename(os.path.dirname(self.audio_path))
+            assert self.predominant_instrument in INST_DICT, "Instrument {} not in instrument dict".format(self.predominant_instrument)
+
+            # Drum presence annotation is present
+            if 'dru' in self._track_paths['audio'][0] or 'nod' in self._track_paths['audio'][0]:
```

Comment:
nit: in __init__, since you use `self._track_paths['audio'][0]` a lot here, maybe create:
`self._audio_filename = self._track_paths['audio'][0]`, and use it everywhere else.